transfers/transfer.py

```diff
@@ -424,9 +464,11 @@ def main(am_user, am_api_key, ss_user, ss_api_key, ts_uuid, ts_path, depth, am_u
     pid_file = get_setting('pidfile', default_pidfile)
     try:
         # Open PID file only if it doesn't exist for read/write
-        f = os.fdopen(os.open(pid_file, os.O_CREAT | os.O_EXCL | os.O_RDWR), 'r+')
+        f = os.fdopen(
+            os.open(pid_file, os.O_CREAT | os.O_EXCL | os.O_RDWR), 'r+')
     except:
-        LOGGER.info('This script is already running. To override this behaviour and start a new run, remove %s', pid_file)
+        LOGGER.info(('This script is already running. To override this '
+                     'behaviour and start a new run, remove %s'), pid_file)
```

Comment:
You don't need the extra set of parens here.

Reply:
:+1: